### PR TITLE
[Relay][Frontend][Torch] fix pytorch frontend not support logical or

### PR DESCRIPTION
**Pull request by taomiao**
add logical_or op to relay pytorch frontend. #16399

### CODE REVIEW

**Comment by taomiao** [comment]
@kevinthesun would you help check this logical_or is ok?

File: python/tvm/relay/frontend/pytorch.py [hunk 1]
```diff
@@ -4237,7 +4243,9 @@ def create_convert_map(self):
             "torchvision::deform_conv2d": self.deform_conv2d,
             "aten::unbind": self.unbind,
             "aten::__and__": self.logical_and,
+            "aten::__or__": self.logical_or,
```
Comment (t-vi) [inline python/tvm/relay/frontend/pytorch.py:4246]: I'm afraid this is not tested and also not quite right.
While `torch.logical_or(torch.tensor(2), torch.tensor(1))` gives `tensor(True)`,
`torch.ops.aten.__or__(torch.tensor(2), torch.tensor(1))` gives `tensor(3)`
(and `torch.tensor(2) or torch.tensor(1))` gives `tensor(2)`, so it is different yet)

**Review by t-vi — CHANGES_REQUESTED** [review]
Thank you for submitting the PR. The `logical_or` looks good, I have doubts about the `__or__` .

File: python/tvm/relay/frontend/pytorch.py (hunk 1 above)
Comment (taomiao) [reply python/tvm/relay/frontend/pytorch.py]: Removed aten::__or__ from the convert_map.

Comment (t-vi) [reply python/tvm/relay/frontend/pytorch.py]: Thank you. Seems good to me now.

**Review by masahi — APPROVED** [review]

**Pull request merged**